back/engines/commercial/bulk_import_ideas/app/views/bulk_import_ideas/web_api/v1/export_form.html.erb

```diff
@@ -130,6 +142,32 @@
               <% end %>
             </div>
 
+          <% elsif field[:format] == :multi_select_image %>
+            <div class="image-select">
+              <% field[:options].each do |option| %>
+                <div>
```

Comment:
Think this `div` is not necessary.

Reply:
Just checked and it is needed as an outer container for each option